extension/lib/bigcommerce/customer/shapeRequestError.js

```diff
@@ -0,0 +1,28 @@
+const BigCommerceRequestClientError = require('./request/ClientError')
+const BigCommerceRequestUnparseableMessageError = require('./request/UnparsableMessageError')
+
+module.exports = function shapeRequestError (err) {
+  // Try parsing the (potential) underlying api error
+  const errorMessageMatch = err.message.match(/({.+})/)
```

Comment:
You can omit the parenthesis here...
```suggestion
  const errorMessageMatch = err.message.match(/{.+}/)
```